docs/DEVELOPERS.md

```diff
@@ -8,9 +8,31 @@ lpac is written with C99 and compatible with [SGP.22 version 2.2.2](https://www.
 
 #### Debian/Ubuntu
 
-With `build-essential` `cmake` `git` `g++` `libpcsclite-dev` `libcurl4-openssl-dev` installed.  
-Run `cmake` and `make`, and then all things will be in `output` directory.  
-If you want to get a Deb package, pass `-DCPACK_GENERATOR=DEB` when invoke `cmake`.
+- Install dependencies and download source code:
```

Comment:
This is documents for **developers**, not normal users.

Reply:
this makes it easier for developers to understand.